examples/running_average/client/running_average.cc

```diff
@@ -84,9 +86,11 @@ int main(int argc, char** argv) {
   // Retrieve average.
   int average_0 = retrieve_average(stub_0.get());
   LOG(INFO) << "client 0 average: " << average_0;
+  assert(average_0 == 100);
```

Comment:
Rather than pulling in a new `#include` and having two different ways to terminate when something's not right, maybe instead:
```C++
if (average_0 != 100) {
  LOG(FATAL) << "Unexpected average " << average_0;
}
```

Reply:
Done